modules/persistence/src/services/entries/index.ts

```diff
@@ -0,0 +1,20 @@
+import { deserialize } from 'bson';
+import { ObjectId } from 'mongodb';
+import { insert } from '../connector';
+
+const COLLECTION_NAME = 'documents';
+
+// Service responsible for memoization of page level documents.
+// Any extraneous logic performed on page level documents as part of upload should be added here
+// or within subfolders of this module
+const entriesFromZip = async (zip) => {
```

Comment:
minor nitpick, can we use a different name than `entries`? seems generic / doesn't align with db name either. documents? astEntries?

Reply:
yeah, this should probably just be called `pages`, don't know why I didn't think of that before 🤔